src/base/static/components/organisms/main-map.tsx

```diff
@@ -135,6 +157,11 @@ const statePropTypes = {
   isMapDraggingOrZooming: PropTypes.bool.isRequired,
   isMapDraggedOrZoomedByUser: PropTypes.bool.isRequired,
   isMapTransitioning: PropTypes.bool.isRequired,
+  isWithMapFilterSlider: PropTypes.bool.isRequired,
```

Comment:
I know most of these checks were previously pushed down to the widgets themselves as a performance optimization in `MainMap`. I relocated them here, though, because I think it simplifies the logic required in hooks in widget components. Since we can't conditionally render widget components before all hooks are declared (i.e. use a line like:
```
if (!mapFilterSliderConfig) {
  return null;
}
```
I was finding it necessary to write a lot of conditional logic in the hooks themselves to prevent reference errors for when `mapFiltersSliderConfig` is `undefined`.